src/main/appOne.java

```diff
@@ -0,0 +1,11 @@
+class Test{
```

Comment:
line no: 2 - Method name should be more descriptive to indicate it is checking for reference equality